spec/integration/output_stream_spec.rb

```diff
@@ -17,7 +17,28 @@
             true
           end
         end
-      "
+      SPEC
+
+      run_command ''
+      expect(last_cmd_stdout).to be_empty
+      in_current_dir do
+        expect(File.read('saved_output')).to include('1 example, 0 failures')
+      end
+    end
+
+    it 'writes to the right output stream even when its a filename' do
+      write_file_formatted 'spec/example_spec.rb', <<-SPEC
+        RSpec.configure do |c|
+          c.formatter = :documentation
+          c.output_stream = 'saved_output'
+        end
+
+        RSpec.describe 'something' do
+          it 'succeeds' do
+            true
+          end
+        end
+      SPEC
```

Comment:
Does it make sense to add a spec for `Pathname` as well, given that's apparently supported (based on your spec below)?